circleguard/gui.py

```diff
@@ -1060,15 +1333,46 @@ def __init__(self):
-                "threshold_cheat", 30)
-        self.threshold_display = SliderBoxSetting("Stealing Display", "Comparisons that score below this will be printed to the console",
-                "threshold_display", 100)
-        self.grid = QVBoxLayout()
-        self.grid.addWidget(self.threshold_steal)
-        self.grid.addWidget(self.threshold_display)
-        self.grid.setAlignment(Qt.AlignTop)
-        self.setLayout(self.grid)
+        self.steal_max_sim = SliderBoxSetting("Max similarity", "ReplaySteal comparisons that score below this "
+                "will be stored so you can view them, and printed to the console", "steal_max_sim", 100)
+        self.steal_max_sim_display = SliderBoxSetting("Max similarity display", "ReplaySteal comparisons that "
+                "score below this will be printed to the console", "steal_max_sim_display", 100)
+        self.relax_max_ur = SliderBoxSetting("Max ur", "Replays that have a ur lower than this will be stored "
+                "so you can view them, and printed to the console", "relax_max_ur", 300)
+        self.relax_max_ur_display = SliderBoxSetting("Max ur display", "Replays with a ur lower than this "
+                "will be printed to the console", "relax_max_ur_display", 300)
+        self.correction_max_angle = SliderBoxSetting("Max angle", "Replays with a set of three points "
+                "making an angle less than this (*and* also satisfying correction_min_distance) will be stored so "
+                "you can view them, and printed to the console.", "correction_max_angle", 360)
+        # display options for correction are more confusing than they're worth, especially
+        # when we don't have a good mechanism for storing Snaps in the Result tab
+        # or visualizer support for the Snap timestamps. TODO potentially add back
+        # if we can provide good support for them.
+        # self.correction_max_angle_display = SliderBoxSetting("Max angle display ", "Replays with a set of "
+        #         "three points making an angle less than this (*and* also satisfying correction_min_distance_display) will be printed "
+        #         "to the console.", "correction_max_angle_display", 360)
+        self.correction_min_distance = SliderBoxSetting("Min distance", "Replays with a set of three points "
+                "where either the distance from AB or BC is greater than this (*and* also satisfying correction_max_angle) "
+                "will be stored so you can view them, and printed to the console.", "correction_min_distance", 100)
+        # self.correction_min_distance_display = SliderBoxSetting("Min distance display", "Replays with a set of "
+        #         "three points where either the distance from AB or BC is greater than this (*and* also satisfying "
+        #         "correction_max_angle_display) will be printed to the console.", "correction_min_distance_display", 100)
```

Comment:
Please clean this up. This can be added when we need it but there is no reason for it to be a comment for now